.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: rust-build
+
+on:
+  push:
+    branches: [ "**" ]
```

Comment:
Maybe on pr is enough, we can protect main branch.

Reply:
LGTM